galaxy/templates/cronjob-maintenance.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: {{ include "galaxy.fullname" . }}-maintenance
+  labels:
+    {{- include "galaxy.labels" . | nindent 4 }}
+spec:
+  schedule: "0 2 * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          securityContext:
+            runAsUser: 101
+            runAsGroup: 101
+            fsGroup: 101
```

Comment:
Should be using Galaxy security context.

Reply:
Done in https://github.com/galaxyproject/galaxy-helm/pull/326/commits/68cfbbcf4102e38be4755e4d0342a1e540e4130d